examples/DancingGoat/Helpers/AreaRestrictionHelper.cs

```diff
@@ -26,7 +26,7 @@ public static string[] GetLandingPageRestrictions()
 
         private static IEnumerable<string> GetWidgetsIdentifiers()
         {
-            return new ComponentDefinitionProvider<WidgetDefinition>()
+            return new Kentico.Builder.Web.Mvc.ComponentDefinitionProvider<WidgetDefinition>()
```

Comment:
Is it necessary to specify namespace in this way? Can we use `using` directive here?